.github/workflows/bump-opbeans-node.yml

```diff
@@ -0,0 +1,37 @@
+---
+## Workflow to periodically check if there is an available newer APM agent version, e.g.
+## "1.2.3". If so, then update to it and tag this repo with that version, e.g.
+## "v1.2.3".
+name: Bump opbeans node
+
+on:
+  schedule:
+    - cron:  '0 20 * * 6'
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+          cache: 'npm'
+      - name: Install Updatecli in the runner
+        uses: updatecli/updatecli-action@v2
```

Comment:
```suggestion
        uses: updatecli/updatecli-action@v3
```

Reply:
I just checked https://github.com/updatecli/updatecli-action/branches and https://github.com/updatecli/updatecli-action/releases.
It seems like there is no `v3`. 

Also referring to https://github.com/elastic/opbeans-node/pull/179#discussion_r1052159629, I guess we should use the SHA since updatecli-action does not seem to be verified.

Reply:
https://github.com/elastic/opbeans-node/pull/179/commits/0cbe25d35d5a70db4fb2244f61835ecfd4884e86

Reply:
The branch v3 do not exist

Reply:
you are true, I grab the version from `actions/checkout` instead `updatecli/updatecli-action`